tests/admin_inlines/admin.py

```diff
@@ -342,6 +344,103 @@ class ClassAdminStackedVertical(admin.ModelAdmin):
 
 
+# admin for #32219 and earlier functionality
+# classes for testing combinations of verbose and verbose_plural
+# settings in models and in inlines.
+class NonVerboseChildVerboseParentInline(admin.TabularInline):
+    model = NonVerboseChild
+    verbose_name = 'Inline Non-Verbose Kid'
+
+
+class VerboseChildVerboseParentInline(admin.TabularInline):
+    model = VerboseChild
+    verbose_name = 'Inline Verbose Kid'
+
+
+class VerbosePluralChildVerboseParentInline(admin.TabularInline):
+    model = VerbosePluralChild
+    verbose_name = 'Inline Verbose Plural Kid'
+
+
+class MultiVerboseChildVerboseParentInline(admin.TabularInline):
+    model = MultiVerboseChild
+    verbose_name = 'Inline Multi-Verbose Kid'
+
+
+class NonVerboseChildVerbosePluralParentInline(admin.TabularInline):
+    model = NonVerboseChild
+    verbose_name_plural = 'Inline Non-Verbose Kids'
+
+
+class VerboseChildVerbosePluralParentInline(admin.TabularInline):
+    model = VerboseChild
+    verbose_name_plural = 'Inline Verbose Kids'
+
+
+class VerbosePluralChildVerbosePluralParentInline(admin.TabularInline):
+    model = VerbosePluralChild
+    verbose_name_plural = 'Inline Verbose Plural Kids'
+
+
+class MultiVerboseChildVerbosePluralParentInline(admin.TabularInline):
+    model = MultiVerboseChild
+    verbose_name_plural = 'Inline Multi-Verbose Kids'
+
+
+class NonVerboseChildMultiVerboseParentInline(admin.TabularInline):
+    model = NonVerboseChild
+    verbose_name = 'Inline Non-Verbose Kid'
+    verbose_name_plural = 'Inline Non-Verbose Kids'
+
+
+class VerboseChildMultiVerboseParentInline(admin.TabularInline):
+    model = VerboseChild
+    verbose_name = 'Inline Verbose Kid'
+    verbose_name_plural = 'Inline Verbose Kids'
+
+
+class VerbosePluralChildMultiVerboseParentInline(admin.TabularInline):
+    model = VerbosePluralChild
+    verbose_name = 'Inline Verbose Plural Kid'
+    verbose_name_plural = 'Inline Verbose Plural Kids'
+
+
+class MultiVerboseChildMultiVerboseParentInline(admin.TabularInline):
+    model = MultiVerboseChild
+    verbose_name = 'Inline Multi-Verbose Kid'
+    verbose_name_plural = 'Inline Multi-Verbose Kids'
+
+
+class VerboseParentAdmin(admin.ModelAdmin):
+
+    inlines = [
+        NonVerboseChildVerboseParentInline,
+        VerboseChildVerboseParentInline,
+        VerbosePluralChildVerboseParentInline,
+        MultiVerboseChildVerboseParentInline,
+    ]
+
+
+class VerbosePluralParentAdmin(admin.ModelAdmin):
+
+    inlines = [
+        NonVerboseChildVerbosePluralParentInline,
+        VerboseChildVerbosePluralParentInline,
+        VerbosePluralChildVerbosePluralParentInline,
+        MultiVerboseChildVerbosePluralParentInline,
+    ]
+
+
+class MultiVerboseParentAdmin(admin.ModelAdmin):
+
+    inlines = [
+        NonVerboseChildMultiVerboseParentInline,
+        VerboseChildMultiVerboseParentInline,
+        VerbosePluralChildMultiVerboseParentInline,
+        MultiVerboseChildMultiVerboseParentInline,
+    ]
```

Comment:
We can define inlines in the `site.register(..., inlines=[....])`.

Reply:
Cool. You're right. I did not think of that. Will do.